pipeline/dags/parallel_batch_download.py

```diff
@@ -81,34 +82,34 @@ def download_binding_batch(batch):
 
                     for dc_identifier in batch:
                         binding_id = utils.binding_id_from_dc(dc_identifier)
+                        base_path = os.path.join(BASE_PATH, utils.binding_download_location(binding_id, set_id))
+                        tmp_base_path = os.path.join(TMPDIR, utils.binding_download_location(binding_id, set_id))
 
                         SaveMetsSFTPOperator(
                             task_id=f"save_mets_{binding_id}",
                             api=api,
                             sftp_client=sftp_client,
                             ssh_client=ssh_client,
-                            tmpdir=TMPDIR,
+                            tmpdir=tmp_base_path,
                             dc_identifier=dc_identifier,
                             base_path=base_path,
-                            file_dir=f"{set_id.replace(':', '_')}/{binding_id}/mets",
+                            file_dir="mets",
                         ).execute(context={})
 
                         SaveAltosSFTPOperator(
                             task_id=f"save_altos_{binding_id}",
-                            mets_path=f"{base_path}/{set_id.replace(':', '_')}/{binding_id}/mets",
+                            mets_path=f"{base_path}/mets",
                             sftp_client=sftp_client,
                             ssh_client=ssh_client,
-                            tmpdir=TMPDIR,
+                            tmpdir=tmp_base_path,
                             dc_identifier=dc_identifier,
                             base_path=base_path,
-                            file_dir=f"{set_id.replace(':', '_')}/{binding_id}/alto",
+                            file_dir="alto",
                         ).execute(context={})
 
-                        ssh_client.exec_command(f"rm -r {TMPDIR}/{binding_id}")
+                        ssh_client.exec_command(f"rm -r {tmp_base_path}")
```

Comment:
Ugh, as we of course should be because we might still be processing `/some/temporary/directory/1/12/123/1235` or something. So as `clear_temp_dir()` takes care of all the then-empty directories, I think this is good as it is.